### DISCUSSION TITLE
Wyze Cam Floodlight

### DISCUSSION THREAD

**Discussion by cirrusflyer** [gtxaspec/wz_mini_hacks#91] [Q&A]
Can we assume since the Wyze Cam Floodlight just uses the v3 camera, that this software will work as usual?

https://www.wyze.com/products/wyze-cam-floodlight

**Comment by gtxaspec** [comment]
@cirrusflyer based on this post https://github.com/gtxaspec/wz_mini_hacks/discussions/81#discussion-4141271 it appears so

**Comment by Replayman** [comment]
So, just tried it today, and with the mini-hacks, I am unable to access Floodlight settings in Wyze app anymore - so unable to change light level, detection area, and all other settings making the floodlight like a regular one.
Don't know yet if it kills also other accessories.

**Reply by gtxaspec** [reply]
If you pull the microsd card, can you verify that the floodlight functionality is available again in the app?

**Reply by Replayman** [reply]
Yes, after removing the SD card the floodlight is working well. I only enabled RTSP from the settings file if that can make a clue. Maybe too few RAM in the cam for both accessories and RTSP processing or the minihack (I think I saw this memory issue in RTSP discussions in Wyze forum). Would have to try other accessories.

**Reply by gtxaspec** [reply]
plenty of ram for both rtsp and other stuff, we even add128mb of virtual memory to help stuff along.  I would try to do a fresh install of wz_mini since other users report it working well with the floodlight , and other accessories (like the spotlight and light bulb socket)

**Comment by dudududodododedede** [comment]
I have a v3 with a floodlight and have had no issues with the wz_mini firmware (aside from the floodlight being flaky but it's always been flaky... need to manually power cycle it every few weeks... it's not a super reliable product)

**Reply by Replayman** [reply]
Thank you. Did you enable the RTSP stream ? Maybe it is an issue with RTSP (I have the same issue with RTSP Beta firmware from Wyze, the accessories are still available in the app, but unable to access Floodlight settings).
Also, do you have the last firmware or an old version ? (Maybe a change in last updates)

**Reply by dudududodododedede** [reply]
yes I am using both the Low and High wz_mini RTSP streams
latest firmware on V3

try a different, shorter USB cable. The spotlight is extremely finicky about the cable and power supply

**Reply by vic4news** [reply]
I am having the same issue with the floodlight. I can no longer access the floodlight settings from the IOS app. The floodlight still works and my shortcuts to turn the light on or off in the app work.

i do have hi res Rtsp for the camera enabled and also set the option to disable firmware updates

**Reply by b1gb33** [reply]
Do you have the spotlight or the floodlight?  Your comment would make me think the Spotlight; but this thread is about the floodlight. The Floodlight has the cable & power supply integrated into the base.  There is no provision to change the USB cord or power supply with the floodlight.  Thanks in advance!

**Comment by b1gb33** [comment]
I have a new floodlight install; v3 camera is 4.36.9.1139 with a fresh current build of wz_mini installed.  Using RTSP.  No floodlight is detected in the Wyze app when the SD card with wz_mini installed inserted.  After checking the #81 thread I and here, don't see any evidence of working floodlight control / configuration while wz_mini is running.  Happy to help troubleshoot.
